26-register_publication/543-creator_begins_wizard_registration_and_navigates_do_description_tab.feature

```diff
@@ -0,0 +1,22 @@
+  Feature: Creator begins Wizard registration and navigates to Description tab
+
+  Scenario: Creator begins Wizard registration and navigates to Description tab
+    Given Creator begins registering a Registration in the Wizard
+    When they navigate to the Description tab
+    Then they see the Description tab is selected
+    And they see mandatory fields for
+      | Title |
+    And they see optional fields for
+      | Abstract                     |
+      | Description                  |
+      | Date published               |
+      | NPI disciplines              |
+      | Keywords                     |
+      | Primary language for content |
+      | Project association          |
+    And they see the tab Reference is clickable
+    And they see the tab Contributor is clickable
+    And they see the tab Files and Licenses is clickable
+    And they see the tab Summary is clickable
+    And they see Next is enabled
+    And they see Save is enabled

```

Comment:
her skulle det kanskje også vært delt opp i flere scenario: https://github.com/BIBSYSDEV/NVA-scenarios/blob/master/26-register_publication/1694-creator_selects_reference_type_student_thesis_and_selects_subtype.feature

Reply:
men er det forskjell på feltene på description tab basert på type da?

Reply:
Tror ikke det skal være det ut i fra min (litt foreldete) erfaring

Reply:
tenkte egentlig mest på skille mellom mandatory og optional, med et scenario for alle felter, og et annet for validering

Reply:
åja, selvfølgelig 🤦  ja - det er nok lurt!

Reply:
Splitter i to scenarioer